### ISSUE TITLE
Lack of Documentation for Translators onboarding

### ISSUE THREAD

**Issue by loicteixeira** [wagtail/wagtail#7115]
We need to ease onboarding of new translators:

- Why translation is important?
- How to get started (account creation)
  - Create an account and then get approved by a Wagtail team member? Does this happen seamlessly or do we need to ping someone?
  - What are the steps required to be a reviewer and not just a translator?
- How to navigate around Transifex
  - A link to this [Wagtail Space Lightning Talk](https://youtu.be/sLI_AuOMUQw?t=17) would be a good start, but we probably want a text version of that and/or a longer format video
- Invite translator to join the corresponding Slack channel
- How to test the translations locally
  - There is the [Managing Wagtail Translations](https://github.com/wagtail/wagtail/wiki/Managing-Wagtail-translations) wiki page, but it’s aimed at contributors.
  - It ties back to the lack of context (see #7116), but it can be hard to know if the translation is correct without being able to check it. Otherwise, we end up waiting for the next version to be released and then wait again for the following version to fix any issues.
- Can each language have its own page to clarify some common decisions?
  - Transifex' Glossary is a good start, but:
    - it isn't really the place to document why a specific decision has been made
    - it has to be tied to a specific term, but some decisions might concern style or grammar in general
  - Would we require such pages to be written in English?
    - Being able to use the target would be simpler for translators, but it would make moderation harder.
  - Should it live with the rest of the documentation? Doing so would mean any change has to go through a PR.

**Comment by banjo-tech** [comment]
Hi @thibaudcolas I will like to work on this

**Comment by lb-** [comment]
@banjo-tech have you used Transifex? It might be good to pick this up if you have had some, even minimal, experience with writing translations.

If so, I recommend the first step would be to write a bit of a draft docs section as a comment, including where you think this should go in the docs. 

Then after a bit of feedback you can prepare a PR 

Also - it would be good to have a read of the Related discussion about how to make sure Wagtail works better across the board for non-english speakers. 

https://github.com/wagtail/wagtail/discussions/8017

**Comment by lb-** [comment]
Bit of confusion about the 'structure' of Transifex - could be good to pull out Matt's response to help with this onboarding page. 

https://github.com/wagtail/wagtail/issues/9378

**Comment by banjo-tech** [comment]
@lb- sorry for the late reply. I have not used transifex. I have however created an account .What I intend to do is make a translation using transifex and then document the process if that is a good idea. I would also pull out Matt's response and go through it. Thank you for the suggestion. When I create a draft I will update this comment before making a pull request

**Comment by lb-** [comment]
That's a great way to go about it. It's also ok to solve 'some' of this PR with a start in the docs.

**Comment by banjo-tech** [comment]
Alright I will thank you

**Comment by banjo-tech** [comment]
[wagtal onbording translators.docx](https://github.com/wagtail/wagtail/files/9806013/wagtal.onbording.translators.docx)

@lb-  I have attach a draft in this comment, please go through it and let me know what you think.

**Comment by lb-** [comment]
@banjo-tech I'm not sure on the approval stuff, when I joined (a while ago) I didn't see that requirement.  

Maybe join the `#translators` chat in Slack and ask there. https://wagtailcms.slack.com/archives/CTKN6UXKN

Also, any chance you can upload a txt file / markdown file instead?

**Comment by banjo-tech** [comment]
[translator onboarding .md](https://github.com/wagtail/wagtail/files/9807782/translator.onboarding.md)
 Thank you @ lb the slack channel was a useful suggestion. 
Please review this draft

**Comment by lb-** [comment]
A few points if feedback on the draft doc

- we use en-GB spelling so `organisation` not `organization` etc (basically remove the z's and replace with s)
- I get the idea of the intro - we appreciate your interest - but I think let's focus this intro on a high level overview. Something like 'Wagtail uses transifex to translate the content for the admin interface. Our goal is ensure that Wagtail can be used by those that speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code' .
- it would be good to add an overview of the translation workflow. Basically each new release we have a two week ish release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated each RC until the final release. If translations reach a threshold of X% (I think it's 90) we communicate that this is an officially supported translation in the docs.

To help with your PR - it would be good for you to have a look at how we write docs. 
- Read our documentation on writing Documentation - for example try to avoid links with the label 'here'. https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html
- https://docs.wagtail.org/en/stable/contributing/documentation_modes.html
- https://docs.wagtail.org/en/stable/contributing/general_guidelines.html


As for docs. We should ensure there is a link to this ne page in this section https://docs.wagtail.org/en/stable/contributing/index.html#translations

And make this new page a sub-page if the contributing section. 

Also it might be good to take some inspiration from Django's docs for this kind of thing. https://docs.djangoproject.com/en/dev/internals/contributing/localizing/


Thanks for putting your ideas out there - this is an awesome start. Looking forward to the Pull Request.

**Comment by banjo-tech** [comment]
Thank you for the thorough review. I will make the necessary corrections and go through the documents

**Comment by banjo-tech** [comment]
@lb-  does the RC in the test above mean required contribution?

**Comment by gasman** [comment]
@banjo-tech RC means release candidate.

**Comment by banjo-tech** [comment]
@lb can you please explain this sentence "Those are then updated each RC until the final release" .
I don't quiet get the message you are trying to pass across

**Comment by lb-** [comment]
@banjo-tech have a read of how the Django docs explains it - our process is essentially the same. 

But basically when we have a release candidate - we will pull in the new translations from transifex into the code. Then we do that again for the final official release. 

Also have a look at the migration of translations instructions. 
https://github.com/wagtail/wagtail/wiki/Managing-Wagtail-translations

Note. This is for developers but this is the process that happens each RC

**Comment by banjo-tech** [comment]
Thank you so much @lb- I think I am done with the document now. 
[internal link to another document](/reference/contrib/legacy_richtext) - I want to be sure if this is how I'd link the new document to the contributing  to wagtail page

**Comment by lb-** [comment]
@banjo-tech I reckon the next step now is to open a PR with your proposed new page.

**Comment by banjo-tech** [comment]
@lb- I will like to open a pull request but I'm unable to link the new file to the index page.

**Comment by lb-** [comment]
Can you reach out on Slack in the Outreachy channel. We can help more there. Just explain what you're doing and the error you have encountered

**Comment by banjo-tech** [comment]
Okay thank you

**Comment by lb-** [comment]
@loicteixeira I will close this as fixed once https://github.com/wagtail/wagtail/pull/9431 is merged as I think it covers most of the original intent of this issue.

Some outstanding items though;

> How to test the translations locally 
I have added this to the https://github.com/wagtail/wagtail/issues/7116 issue.

> Can each language have its own page to clarify some common decisions?

Not sure if we should split this to its own issue, I think it needs a bit of a discussion though, maybe it is best to be added to https://github.com/wagtail/wagtail/discussions/8017 ?

**Comment by banjo-tech** [comment]
Hi @lb- Thank you for the review on the previous PR. I will like to work on the remaining part of this issue but I guess I will have to wait on a response from @loicteixeira or could I go ahead without creating an issue first?

**Comment by loicteixeira** [comment]
Thank you both for moving this along 🙏 

> How to test the translations locally
> I have added this to the https://github.com/wagtail/wagtail/issues/7116 issue.

Yeah, it makes sense to move it to the other issue. I'll also comment there though.

> I will like to work on the remaining part of this issue but I guess I will have to wait on a response from @loicteixeira or could I go ahead without creating an issue first?

Sorry I haven't been more available, but don't let me block you.
That being said, I'm not sure what needs a new issue? @lb- added the remaining points of this issue to https://github.com/wagtail/wagtail/issues/7116, but if this is something else, please create an issue (or comment on https://github.com/wagtail/wagtail/issues/7116 if that's related and we will update the top comment accordingly).

> Can each language have its own page to clarify some common decisions?
> Not sure if we should split this to its own issue, I think it needs a bit of a discussion though, maybe it is best to be added to https://github.com/wagtail/wagtail/discussions/8017 ?

I'm torn on this one. On one hand, it should be with the documentation (separate page though) so anybody can update it, but I'm not sure all translators would want to/be able to go through the whole clone/edit/commit/pull-request process. On the other hand, GitHub discussions would be a nice free form thing, but it's hard to "pin" decisions, and would probably require to update the top level comment, which can only be done by the author of that comment. So yeah, it probably makes sense to discuss this futher on https://github.com/wagtail/wagtail/discussions/8017.

**Comment by lb-** [comment]
Sounds good @loicteixeira  - yeah the 'glossary' thing is good but agree it needs to be translated, it may also provide a clear 'home' for naming conflicts like https://github.com/wagtail/wagtail/issues/9201

But there is also a balance between this and expecting translators to have some knowledge of how Wagtail works.

@banjo-tech based on this discussion so far, I do not think there is anything else to be done under this issue. But I will comment on https://github.com/wagtail/wagtail/issues/7116

**Comment by banjo-tech** [comment]
Thank you @lb- and @loicteixeira I will read through #7116 and see if the issue is something I can work on
